pkg/kubelet/network/exec/exec.go

```diff
@@ -29,6 +29,31 @@ limitations under the License.
 //         - setup, called after the infra container of a pod is
 //                created, but before other containers of the pod are created
 //         - teardown, called before the pod infra container is killed
+//         - status, called at regular intervals and is supposed to return a json
+//                formatted output indicating the pod's IPAddresses. An empty string value or an erroneous output
+//                will mean the container runtime (docker) will be asked for the PodIP
+//                e.g. {
+//                         "apiVersion" : "latest",
```

Comment:
let's start with v1beta1 and we can sync it to the main API when we mark this non-alpha